src/utils/enums.ts

```diff
@@ -0,0 +1,4 @@
+export enum AccountDropdown {
+  Version1 = 1,
+  Version2 = 2,
+}
```

Comment:
This is strictly related to the `AccountStatus` component and should be exported from there